src/react-integration/NetworkErrorBoundary.tsx

```diff
@@ -5,7 +5,7 @@ export interface NetworkError extends Error {
   response?: { statusText?: string; body?: any };
 }
 
-function isNetworkError(error: NetworkError | any): error is NetworkError {
+function isNetworkError(error: NetworkError | unknown): error is NetworkError {
```

Comment:
unrelated - just snuck this in

Reply:
![sneaky](https://media3.giphy.com/media/1eEH72j9Cd0VEFosRf/giphy.gif)